code/game/world.dm

```diff
@@ -7,6 +7,78 @@
 
 GLOBAL_VAR(restart_counter)
 
+/**
+ * WORLD INITIALIZATION
+ * THIS IS THE INIT ORDER:
+ *
+ * BYOND =>
+ * - (secret init native) =>
+ *   - world.RealGlobalsPreInit() =>
+ *     - world.init_byond_tracy()
```

Comment:
Again FYI @LemonInTheDark is changing this to run tracy as soon as possible, so that we can profile both global init time *and* time to create statics

Reply:
This _IS_ profiling the global init time. There is no faster way to call this (other than inlining `init_byond_tracy`).

Reply:
Which I did